connectors-home/modules/ROOT/pages/windows-gateway-services/windows-gateway-services-guide-mule4.adoc

```diff
@@ -146,6 +146,23 @@ In the following table you can find the configuration settings that are only rel
 
 When your queue is marked to require authentication, you can impersonate the caller user as specified in the configuration settings. Additionally, if you work with a remote queue, the connector has a particular header to override this behavior.
 
+[[work-with-remote-queues]]
+=== Work with Remote Queues
+
+If the Anypoint Gateway service runs on one machine and must access a remote queue on another machine, you must do one of these options (based on your environment configuration):
+
+* Verify machine grouping
++
+. Ensure that both machines belong to the same workgroup.
+. If anonymous access is not enabled, grant full access to the `Anonymous` user on the remote queue.
+. Verify that the `GET` operation works successfully under these updated anonymous access settings.
```

Comment:
```suggestion
. Verify that the *GET* operation works successfully under these updated anonymous access settings.
```

Reply:
According to our style guide, operations should be bolded: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=with%20blockout.-,bold%20font,-Apply%20bold%20font